specs/merge/validator.md

```diff
@@ -39,38 +42,73 @@ Please see related Beacon Chain doc before continuing and use them as a referenc
 
 | Name | SSZ equivalent | Description |
 | - | - | - |
-| `PayloadId` | `uint64` | Identifier of a payload building process |
+| `PayloadId` | `Bytes8` | Identifier of a payload building process |
 
-## Protocols
+## Helpers
 
-### `ExecutionEngine`
+### `get_pow_block_at_terminal_total_difficulty`
```

Comment:
Moved `get_pow_block_at_terminal_total_difficulty` and `get_terminal_pow_block` up to helpers section.

Not changes made to contents of the functions.